tests/db_functions/test_chr.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import IntegerField
+from django.db.models.functions import Chr, Left, Ord
+from django.test import TestCase
+
+from .models import Author
+
+
+class ChrTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.create(name='John Smith', alias='smithj')
+        Author.objects.create(name='Élena Jordan', alias='elena')
+        Author.objects.create(name='Rhonda')
+
+    def test_chr(self):
+        authors = Author.objects.annotate(first_initial=Left('name', 1))
+
```

Comment:
You can remove many of the blank lines in the tests. The tests are short enough that they're not adding any readability.

Reply:
@timgraham, I have minimized the blank lines as much as possible as PEP 8 recommends. PyCharm also warns me when I remove extra blank lines in violation of PEP 8.  My blank lines in the tests seem to match the spacing of other simple tests as well.  Please let me know how you would like me to proceed.